recipes/fast-dds/all/conanfile.py

```diff
@@ -83,19 +75,13 @@ def requirements(self):
             self.requires("openssl/[>=1.1 <4]")
 
     def validate(self):
-        if self.settings.compiler.cppstd:
-            check_min_cppstd(self, self._min_cppstd)
-        check_min_vs(self, "192")
-        if not is_msvc(self):
-            minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
-            if minimum_version and Version(self.settings.compiler.version) < minimum_version:
-                raise ConanInvalidConfiguration(
-                    f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
-                )
```

Comment:
The reason for these checks to exist is because Conan 1 does not properly model the cppstd of compilers, so we need a way to filter unsupported compilers. Conan 1 will stop being supported at some point in the future, but for now we still need to keep it around in CCI, meaning the check can not complety go away!

I'd go with the route of relaxing the gcc version if older ones do work :)